src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do list</title>
+</head>
+
+<body>
+    <main>
+        <section class="container">
+            <div class="ctn-head">
+                <h5 class="head-title">Today's To Do</h5>
+                <div class="ctn-icon"><i class="fa-solid fa-arrows-rotate"></i></div>
+            </div>
+            <div class="line"></div>
```

Comment:
- [ ] As required in HTML and CSS best pratique, it will be better if we use the `<hr>` tag for the line instead of `<div>` on line 17. What do you think?

Reply:
this has been implemented thank you